docker/docker-compose-production.yml

```diff
@@ -2,179 +2,167 @@ version: '2'
       - stb-frontend
     command: gunicorn stroyprombeton.wsgi:application -c /etc/gunicorn.py -b 0.0.0.0:$VIRTUAL_HOST_STAGE_PORT
 
-  stb-celery-beat:
+  celery-beat:
     image: fidals/stb:prod
     container_name: stb-celery-beat
     restart: always
-    environment:
-      - DJANGO_SETTINGS_MODULE=$DJANGO_SETTINGS_MODULE
-      - DATABASE_URL=postgres://$DB_USER:$DB_PASS@stb-postgres/$DB_NAME
-      - BROCKER_URL=amqp://$RABBITMQ_DEFAULT_USER:$RABBITMQ_DEFAULT_PASS@stb-rabbitmq:5672/
+    env_file:
+      - env_files/app
+      - env_files/credentials
+      - env_files/paths
     depends_on:
-      - stb-python
-    volumes_from:
-      - stb-source
+      - app
     networks:
       - stb-backend
     command: celery -A stroyprombeton beat -l info
 
-  stb-celery-mail-worker:
+  celery-default-worker:
     image: fidals/stb:prod
-    container_name: stb-celery-mail-worker
+    container_name: stb-celery-default-worker
     restart: always
-    environment:
-      - DJANGO_SETTINGS_MODULE=$DJANGO_SETTINGS_MODULE
-      - DATABASE_URL=postgres://$DB_USER:$DB_PASS@stb-postgres/$DB_NAME
-      - BROCKER_URL=amqp://$RABBITMQ_DEFAULT_USER:$RABBITMQ_DEFAULT_PASS@stb-rabbitmq:5672/
-      - EMAIL_HOST=$EMAIL_HOST
-      - EMAIL_HOST_PASSWORD=$EMAIL_HOST_PASSWORD
-      - EMAIL_HOST_USER=$EMAIL_HOST_USER
-      - EMAIL_PORT=$EMAIL_PORT
-      - EMAIL_SENDER=$EMAIL_SENDER
-      - EMAIL_USE_SSL=$EMAIL_USE_SSL
-      - EMAIL_RECIPIENTS=$EMAIL_RECIPIENTS
-      - SHOP_EMAIL=$SHOP_EMAIL
+    env_file:
+      - env_files/app
+      - env_files/credentials
+      - env_files/paths
     depends_on:
-      - stb-python
-    volumes_from:
-      - stb-source
+      - app
     networks:
       - stb-backend
-    command: celery -A stroyprombeton worker -E -n worker.mail@%%h -l info -Q mail
+    command: celery -A stroyprombeton worker -E -n worker.default@%%h -l info -Q default
 
-  stb-celery-command-worker:
+  celery-default-worker:
     image: fidals/stb:prod
-    container_name: stb-celery-command-worker
+    container_name: stb-celery-default-worker
     restart: always
-    environment:
-      - DJANGO_SETTINGS_MODULE=$DJANGO_SETTINGS_MODULE
-      - DATABASE_URL=postgres://$DB_USER:$DB_PASS@stb-postgres/$DB_NAME
-      - BROCKER_URL=amqp://$RABBITMQ_DEFAULT_USER:$RABBITMQ_DEFAULT_PASS@stb-rabbitmq:5672/
+    env_file:
+      - env_files/app
+      - env_files/credentials
+      - env_files/paths
     depends_on:
-      - stb-python
-    volumes_from:
-      - stb-source
+      - app
     networks:
       - stb-backend
-    command: celery -A stroyprombeton worker -E -n worker.command@%%h -l info -Q command
+    command: celery -A stroyprombeton worker -E -n worker.default@%%h -l info -Q default
 
-  stb-nginx:
-    build:
-      context: ..
-      dockerfile: docker/nginx/Dockerfile
+  nginx:
     image: fidals/stb-nginx:prod
     container_name: stb-nginx
     restart: always
+    env_file:
+      - env_files/ports
     depends_on:
```

Comment:
It is unnecessary, because `volumes_from` will be sufficient

Reply:
TODO